irr/src/CMakeLists.txt

```diff
@@ -213,11 +213,14 @@ if(ENABLE_GLES2)
 	find_package(OpenGLES2 REQUIRED)
 endif()
 if(ENABLE_OPENGL)
-	find_package(OpenGL REQUIRED)
-	# we need EGL so use the import target
-	if(NOT DEFINED EGL_LIBRARY)
+	if(DEVICE STREQUAL "X11")
```

Comment:
are u sure? This doesn't look right. Shouldn't be related to X11 only

Reply:
"X11" here means CIrrDeviceLinux, which is the only thing using GLX.